From d15caf2484548e3f6e06a38e91e3fb2707d270d3 Mon Sep 17 00:00:00 2001
From: Felipe Borges <felipeborges@gnome.org>
Date: Tue, 10 May 2016 14:23:12 +0200
Subject: [PATCH 08/28] mouse: allow focus on switches in the Touchpad section

Enables the navigation/focus on the switches in the Touchpad
section.

https://bugzilla.gnome.org/show_bug.cgi?id=766186
---
 panels/mouse/gnome-mouse-properties.ui | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/panels/mouse/gnome-mouse-properties.ui b/panels/mouse/gnome-mouse-properties.ui
index 25e51c8..05b5030 100644
--- a/panels/mouse/gnome-mouse-properties.ui
+++ b/panels/mouse/gnome-mouse-properties.ui
@@ -332,7 +332,7 @@
         </child>
         <child>
           <object class="GtkFrame" id="touchpad-frame">
-            <property name="visible">False</property>
+            <property name="visible">True</property>
             <property name="can_focus">False</property>
             <property name="margin_top">12</property>
             <property name="margin_bottom">12</property>
@@ -359,7 +359,7 @@
                 <child>
                   <object class="GtkBox" id="touchpad-box">
                     <property name="visible">True</property>
-                    <property name="can_focus">True</property>
+                    <property name="can_focus">False</property>
                     <property name="orientation">vertical</property>
                     <child>
                       <object class="GtkListBox" id="touchpad-listbox">
-- 
2.9.3

